### PR TITLE
Added $ELASTICSEARCH variable

### LINKED ISSUE #2: Forcing linking with Elasticsearch
Hi,

I'm not sure it is a good idea to force the linking of an elasticsearch container. It forces single a point of failure and makes usage of the kibana container hard with clusering tools like mesos.

Why not at least allow to overwrite the ELASTICSEARCH_PORT_9200_TCP variable, instead of forcing the value?

### PR DESCRIPTION
**Pull request by Gingonic**
Added $ELASTICSEARCH variable support to avoid linkage

Closes #2

### CODE REVIEW

**Comment by Gingonic** [comment]
Should fix #2

**Comment by Kosta-Github** [comment]
I ran into that same issue at the very same time... :-)

I want to use it with `mesos` and `marathon` on a compute cluster.

+1 for this fix. would help a lot!

**Comment by Kosta-Github** [comment]
I tested this change and it works for me. No container linking required. Thx!

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -9,7 +9,9 @@ fi
 
 # Run as user "kibana" if the command is "kibana"
 if [ "$1" = 'kibana' ]; then
-	if [ "$ELASTICSEARCH_PORT_9200_TCP" ]; then
+	if [ "$ELASTICSEARCH" ]; then
```
Comment (tianon) [inline docker-entrypoint.sh]: Can you please switch this to `ELASTICSEARCH_URL`? :+1:

File: docker-entrypoint.sh [hunk 2]
```diff
@@ -9,7 +9,9 @@ fi
...
+		sed -ri "s!^(elasticsearch_url:).*!\1 '$ELASTICSEARCH'!" /opt/kibana/config/kibana.yml
+	elif [ "$ELASTICSEARCH_PORT_9200_TCP" ]; then
 		sed -ri 's!^(elasticsearch_url:).*!\1 "http://elasticsearch:9200"!' /opt/kibana/config/kibana.yml
 	else
 		echo >&2 'warning: missing ELASTICSEARCH_PORT_9200_TCP'
```
Comment (yosifkit) [inline docker-entrypoint.sh]: This should probably have `warning: missing ELASTICSEARCH_URL or ELASTICSEARCH_PORT_9200_TCP`

**Comment by Gingonic** [comment]
Yeah $ELASTICSEARCH_URL makes more sense.Changed it and added a comment line for the default case

**Comment by tianon** [comment]
LGTM

**Comment by yosifkit** [comment]
LGTM

**Pull request merged**